timestamps/1142.csv

```diff
@@ -0,0 +1,6 @@
+start,end,title
```

Comment:
Given the way you're splitting, does it not read this line as input timestamps (and give an error?)

Reply:
Nope, because I run:

```python
header = file.readline()
```
which skips the first line of the file. That exists just in case we want to verify if the header is correct in the future, if for example, our timestamps files get more complicated.

Reply:
Neat all good then!